qemu/tests/migration_with_speed_measurement.py

```diff
@@ -87,6 +87,8 @@ def get_migration_statistic(vm):
         last_transfer_mem = 0
         transfered_mem = 0
         mig_stat = Statistic()
+        while vm.monitor.get_migrate_progress() == 0:
+            pass
```

Comment:
Suggest `time.sleep(0.1)` to let cpu relaxes.

Reply:
I though about it as well, but at least on my laptop the initialization is so quick it was unnecessarily slowing the execution. Let's do 2/3 on reviews, I'm slightly inclined towards no sleep as the busy loop involves already monitor and other places where IO takes place, I take your input as preferring timeout so let's see who else will add an opinion...

Reply:
If monitor and IO operations eats enough time I vote to keep it as is.